package.json

```diff
@@ -24,6 +24,8 @@
     "fast-glob": "^3.2.11",
     "graphql": "^16.5.0",
     "graphql-tag": "^2.12.6",
+    "isomorphic-unfetch": "^3.1.0",
```

Comment:
This package is not needed if you use Next.js and can actually cause issues. Next.js provides its own own polyfill for fetch